pkg/parser/additional/additional.go

```diff
@@ -0,0 +1,199 @@
+package additional
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+	"gopkg.in/yaml.v2"
+)
+
+func checkConvertingError(err error) {
+	if err != nil {
+		log.Error().Msg("Failed to convert 'string' into 'int'")
+	}
+}
+
+func getRegexMatch(regex, text string) string {
+	var re = regexp.MustCompile(regex)
+	clean := re.FindAllString(text, -1)
+
+	if clean == nil {
+		return ""
+	}
+
+	return clean[0]
+}
+
+func getExpirationDate(datetime string) [3]int {
+	// get notAfter datetime
+	expirationInfo := getRegexMatch(`(notAfter=)[a-zA-Z]+[ ]+[0-9]{1,2}[ ]([0-9]{2}:[0-9]{2}:[0-9]{2})[ ][0-9]{4}`, datetime)
+	clean := strings.Split(expirationInfo, "=")
+
+	months := map[string]int{
```

Comment:
I'm not sure if you have to do this. Maybe there's a function under the golang's time package to parse abbreviated months.